Flaky unit test: ReaderGroupNotificationTest.testSegmentNotifications

```
io.pravega.test.integration.ReaderGroupNotificationTest > testSegmentNotifications FAILED
    java.lang.AssertionError: expected:<data2> but was:<null>
        at org.junit.Assert.fail(Assert.java:88)
        at org.junit.Assert.failNotEquals(Assert.java:834)
        at org.junit.Assert.assertEquals(Assert.java:118)
        at org.junit.Assert.assertEquals(Assert.java:144)
        at io.pravega.test.integration.ReaderGroupNotificationTest.testSegmentNotifications(ReaderGroupNotificationTest.java:168)
```

https://travis-ci.org/pravega/pravega/builds/289189284?utm_source=github_status&utm_medium=notification

--- a/test/integration/src/test/java/io/pravega/test/integration/ReaderGroupNotificationTest.java
+++ b/test/integration/src/test/java/io/pravega/test/integration/ReaderGroupNotificationTest.java
@@ -144,7 +144,7 @@ public class ReaderGroupNotificationTest {
         ReaderGroupManager groupManager = new ReaderGroupManagerImpl("test", controller, clientFactory,
                 connectionFactory);
         ReaderGroup readerGroup = groupManager.createReaderGroup("reader", ReaderGroupConfig
-                .builder().build(), Collections
+                .builder().automaticCheckpointIntervalMillis(-1).build(), Collections
                 .singleton("test"));
         @Cleanup
         EventStreamReader<String> reader1 = clientFactory.createReader("readerId", "reader", new JavaSerializer<>(),

--- a/test/integration/src/test/java/io/pravega/test/integration/endtoendtest/EndToEndWithScaleTest.java
+++ b/test/integration/src/test/java/io/pravega/test/integration/endtoendtest/EndToEndWithScaleTest.java
@@ -124,8 +124,10 @@ public class EndToEndWithScaleTest {
         assertTrue(result);
         writer.writeEvent("0", "txntest2").get();
         @Cleanup
-        ReaderGroupManager groupManager = new ReaderGroupManagerImpl("test", controller, clientFactory, connectionFactory);
-        groupManager.createReaderGroup("reader", ReaderGroupConfig.builder().build(), Collections.singleton("test"));
+        ReaderGroupManager groupManager = new ReaderGroupManagerImpl("test", controller, clientFactory,
+                connectionFactory);
+        groupManager.createReaderGroup("reader", ReaderGroupConfig.builder().automaticCheckpointIntervalMillis(-1).
+                build(), Collections.singleton("test"));
         @Cleanup
         EventStreamReader<String> reader = clientFactory.createReader("readerId", "reader", new JavaSerializer<>(),
                 ReaderConfig.builder().build());

